src/ConnectedMode/Migration/ConnectedModeMigration.cs

```diff
@@ -125,10 +128,11 @@ private async Task MigrateImplAsync(BoundSonarQubeProject oldBinding, IProgress<
                 logger.WriteLine(MigrationStrings.Process_SkippingChecking);
             }
 
-            // TODO - trigger unintrusive binding process (will need the binding arguments)
             progress?.Report(new MigrationProgress(0, 1, "TODO: Create new binding", true));
             logger.WriteLine(MigrationStrings.Process_ProcessingNewBinding);
 
+            await unintrusiveBindingController.BindAsync(oldBinding, token);
```

Comment:
One thing we should discuss later is that right now the progress will not be updated in the UI until this gets hit. For me it just showed the first report -> Getting files.. and then nothing up until the code goes into this method at which point all the following logs show. Is this because of the threading or?